qwoyn/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyn",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
```

Comment:
Please register this prefix to slip173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md